sparse_strips/vello_cpu/src/lib.rs

```diff
@@ -4,3 +4,15 @@
 //! This crate implements a CPU-based renderer, optimized for SIMD and multithreaded execution.
 //! It is optimized for CPU-bound workloads and serves as a standalone renderer for systems
 //! without GPU acceleration.
+
+#![expect(
+    clippy::cast_possible_truncation,
+    reason = "We cast u16s to u8 in various places where we\
```

Comment:
I don't think you need the backslash, the reason should be (and is) short enough to fit on one line.